modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,49 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling mistakes.
+:keywords: autocorrect, capitalization, spell, spelling, spellcheck
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tiertwo
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+The {pluginname} plugin automatically corrects a specified set of spelling errors, capitalization errors, or both, as they are being typed.
+
+== Interactive Example
+
+liveDemo::autocorrect[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'autocorrect'
+});
+----
+
+The {pluginname} plugin has a few limitations or known issues that should be noted:
+
+* Currently, only the English language is supported.
+* With both autocorrect and capitalization enabled, when a misspelt word word is typed at the beginning of a sentence, the word is corrected but not capitalized.
```

Comment:
This should not be under the interactive example heading... it should either be before this or in it's on section with it's own heading.